lxd/events.go

```diff
@@ -35,7 +36,13 @@ type eventsServe struct {
 
 // Render starts event socket.
 func (r *eventsServe) Render(w http.ResponseWriter, req *http.Request) error {
-	return eventsSocket(r.s, req, w)
+	err := eventsSocket(r.s, req, w)
+
+	if err == nil {
+		request.MetricsCallback(req, metrics.Success)
+	}
+
+	return err
```

Comment:
what happens to the ongoing metrics record if there is an error here?

Shouldnt request.MetricsCallback always be called passing in the correct result?